chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java

```diff
@@ -254,13 +243,17 @@ public StatisticsGenerator generate() throws IOException {
             persistAggregatedMetricValue(Math.rint(globalStatisticsCalc.getMean()) / 1000D, latencyDesc);
             persistAggregatedMetricValue(Math.rint(globalStatisticsCalc.getStandardDeviation()) / 1000D, latencyStdDevDesc);
 
-            Long startTime = aggregationInfo.getMinTime();
-            Long endTime = aggregationInfo.getMaxTime();
+            Long startTime = aggregationInfo.getStartTime();
+            Long endTime = aggregationInfo.getEndTime();
+
+            log.info("qwe startTime: [{}], endTime: [{}]", startTime, endTime);
```

Comment:
what is qwe? please make logs more informative. why do we need it on the info level?

Reply:
fixed